nbconvert/exporters/tests/test_pdf.py

```diff
@@ -28,8 +28,8 @@ def test_constructor(self):
         self.exporter_class()
 
 
-    @dec.onlyif_cmds_exist('xelatex')
-    @dec.onlyif_cmds_exist('pandoc')
+    @onlyif_cmds_exist('xelatex')
+    @onlyif_cmds_exist('pandoc')
```

Comment:
This could be combined into one decorator, right?

Reply:
yep! good call!